Application/People/Meta/Student/Service/Data/Transfer.php

```diff
@@ -116,8 +116,8 @@ public function createStudentTransfer(
         TblCompany $tblStateCompany = null,
         TblType $tblType = null,
         TblCourse $tblCourse = null,
-        $TransferDate,
-        $Remark,
+        $TransferDate = '',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$TransferDate" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)